spec/features/staff/users_can_view_an_activity_as_xml_spec.rb

```diff
@@ -42,6 +42,27 @@
         end
       end
 
+      context "when the activity does not have actual dates (optional dates)" do
+        let(:activity) {
+          create(:fund_activity,
+            organisation: organisation,
+            identifier: "IND-ENT-IFIER",
+            actual_start_date: nil,
+            actual_end_date: nil)
+        }
+        let(:activity_presenter) { ActivityXmlPresenter.new(activity) }
```

Comment:
Do we need this? I can't see where we use it?

Reply:
I wasn't sure whether I need the activity xml presenter. I thought since we are checking if the info is displayed in the xml we would need it, but that's due to my lack of understanding with presenters (I think I mentioned this to you in the past). Thanks for the suggestion

Reply:
No worries, as a general rule-of-thumb we only want 'just enough' to get the test setup. 

As this is a feature test it is excercising the whole stack, we are making a request, returning an entire page of xml and then looking for a specific element, that process will take care of any presenter needed so you shouldn't have to.